### PR TITLE
certbot deps: python-josepy, python-pyrfc3339, python-acme, python-parsedatetime

### PR DESCRIPTION
**Pull request by a16bitsysop**

### CODE REVIEW

File: contrib/python-acme/template.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "python-requests",
+    "python-pyrfc3339",
+]
+checkdepends = [
+    "python-pytest",
+] + depends
+pkgdesc = "ACME protocol implementation"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "Apache-2.0"
+url = "https://github.com/certbot/acme"
```
Comment (nekopsykose) [inline contrib/python-acme/template.py:25]: this url is a 404

File: contrib/python-josepy/template.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+pkgname = "python-josepy"
+pkgver = "1.14.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-poetry-core",
+    "python-wheel",
```
Comment (nekopsykose) [inline contrib/python-josepy/template.py:9]: don't need wheel here pretty sure

File: contrib/python-acme/template.py (hunk 1 above)
Comment (a16bitsysop) [reply contrib/python-acme/template.py:25]: ah, probably should be https://github.com/certbot/certbot/tree/master/acme ?
As its inside the certbot project

**Comment by nekopsykose** [comment]
also rebase this

File: contrib/python-acme/template.py (hunk 1 above)
Comment (nekopsykose) [reply contrib/python-acme/template.py:25]: hmmm, sure, can't think of anything better

File: contrib/python-acme/template.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "python-installer",
+    "python-wheel",
+]
+depends = [
+    "python-cryptography",
+    "python-josepy",
+    "python-openssl",
+    "python-pytz",
+    "python-requests",
+    "python-pyrfc3339",
```
Comment (nekopsykose) [inline contrib/python-acme/template.py:17]: sort

Comment (a16bitsysop) [reply contrib/python-acme/template.py]: okay

File: contrib/python-josepy/template.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+pkgname = "python-josepy"
+pkgver = "1.14.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-poetry-core",
+]
+depends = ["python-cryptography", "python-openssl", "python-setuptools"]
```
Comment (nekopsykose) [inline contrib/python-josepy/template.py:10]: this doesn't depend on setuptools

Comment (a16bitsysop) [reply contrib/python-josepy/template.py:10]: okay will remove

File: contrib/python-acme/template.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    "python-wheel",
+]
+depends = [
+    "python-cryptography",
+    "python-josepy",
+    "python-openssl",
+    "python-pyrfc3339",
+    "python-pytz",
+    "python-requests",
+    "python-setuptools",
```
Comment (nekopsykose) [inline contrib/python-acme/template.py:18]: this belongs in hostmakedepends

Comment (a16bitsysop) [reply contrib/python-acme/template.py:18]: its in install requires in setup.py

https://github.com/certbot/certbot/blob/2a41402f2ade41b41532e11f29b295eefc3e1552/acme/setup.py#L8

Comment (nekopsykose) [reply contrib/python-acme/template.py:18]: the code doesn't import setuptools or pkg_resources though

Comment (a16bitsysop) [reply contrib/python-acme/template.py:18]: okay will move it

**Pull request merged**